keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Gaussian.java

```diff
@@ -9,51 +9,61 @@
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier,
+ * ARL-TR-2168 March 2000,
+ * 5.1.16 page 29"
+ */
 public class Gaussian implements ContinuousDistribution {
 
-    public static final double SQRT_2PI = Math.sqrt(Math.PI * 2);
-    public static final double LN_SQRT_2PI = Math.log(SQRT_2PI);
-    private final DoubleTensor mu;
-    private final DoubleTensor sigma;
+    private static final double SQRT_2PI = Math.sqrt(Math.PI * 2);
+    static final double LN_SQRT_2PI = Math.log(SQRT_2PI);
+    private final DoubleTensor location;
+    private final DoubleTensor scale;
```

Comment:
I don't like `scale` here because its ambiguous if that means the variance or the standard deviation.

I'd vote for keeping it as `mu` and `sigma` since they are a universal convention, but if we do change it then it ought to be to `mean` and `variance`.

Reply:
Good point! Maybe we will have to have special cases for Gaussian ...